azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/utils/ResourceNamer.java

```diff
@@ -63,13 +63,13 @@ public String randomUuid() {
     }
 
     private String randomString(int length) {
-        String str = "";
+        StringBuilder str = new StringBuilder();
         while (str.length() < length) {
```

Comment:
Preferring stringBuilder append over string concat

Reply:
[P3]: Behavior of `str +=` and `str.append` are same. I don't know which one is better. fine with both.